R/lat_bins_equal.R

```diff
@@ -0,0 +1,114 @@
+#' Generate equal-area latitudinal bins
+#'
+#' A function to generate approximately equal-area latitudinal bins for a
+#' user-specified given number of bins and latitudinal range. This approach is
+#' based on calculating the curved surface area of spherical segments bounded
+#' by two parallel disks.
+#'
+#' @param n \code{numeric}. A single numeric value defining the number of
+#'   equal-area latitudinal bins to split the latitudinal range by (as defined
+#'   by `min` and `max`).
+#' @param min \code{numeric}. A single numeric value defining the lower limit
+#'   of the latitudinal range (defaults to -90).
+#' @param max \code{numeric}. A single numeric value defining the upper limit
+#'   of the latitudinal range (defaults to 90).
+#' @param r \code{numeric}. The radius of the Earth in kilometres. Defaults to
+#'   the the mean radius of the Earth (6371 km).
+#' @param plot \code{logical}. Should a plot of the latitudinal bins be
+#'   generated?
+#' @return A \code{data.frame} of user-defined number of latitudinal bins.
```

Comment:
It would be good to add the column names here, I think. For completeness should we also add e.g. "if plot = TRUE, a plot is also returned." ?